Orange/widgets/utils/owlearnerwidget.py

```diff
@@ -123,7 +123,7 @@ def __init__(self):
         self.preprocessors = None
         self.outdated_settings = False
         self.setup_layout()
-        QTimer.singleShot(0, self.apply)
+        QTimer.singleShot(0, self.update_learner)
```

Comment:
'this problem' = the problem with linear regression, not the problem fixed by this PR here.